### PR TITLE
[Refactor]:change config content

### PR DESCRIPTION
**Pull request by wanghonglie**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
repair config content

## Modification
Fix file dependency path　in README.md; metafile.yaml; ${CONFIG}.py

### CODE REVIEW

File: configs/atss/atss_r101_fpn_8xb8-amp-lsj-200e_coco.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-_base_ = './atss_r50_fpn_lsj_200e_8x8_fp16_coco.py'
```
Comment (chhluo) [inline configs/atss/atss_r101_fpn_8xb8-amp-lsj-200e_coco.py:1]: metafile and readme

File: configs/carafe/metafile.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ Collections:
 Models:
   - Name: faster_rcnn_r50_fpn_carafe_1x_coco
```
Comment (chhluo) [inline configs/carafe/metafile.yml:23]: Name should be modified.

File: configs/cascade_rcnn/cascade-mask-rcnn_r50_fpn_ms-3x_coco.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
 _base_ = [
-    '../common/mstrain_3x_coco_instance.py',
-    '../_base_/models/cascade_mask_rcnn_r50_fpn.py'
+    '../common/ms_3x_coco_instance.py',
```
Comment (chhluo) [inline configs/cascade_rcnn/cascade-mask-rcnn_r50_fpn_ms-3x_coco.py:2]: image

why `ms_3x_coco_instance.py`? Should be `ms-3x_coco-instance`?

**Comment by BIGWangYuDong** [comment]
We should also fix UT, in [tests/model/detectors](https://github.com/open-mmlab/mmdetection/tree/refactor-cfg/tests/test_models/test_detectors)

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**